docs/pow.md

```diff
@@ -0,0 +1,71 @@
+# Proof-of-work in the faucet
+
+The faucet uses POW challenges for anti-dos and dynamically shifts the
+difficulty based on the faucet's balance.
+
+Generally, the less bitcoin in the faucet, the more difficult it is to get out
+via solving the POW challenge.
+
+This was initially implemented via a logarithmic based function but since the
+difficulty parameter already exponentially increases the difficulty, we switched
+to a linear approach which is easier to read and is less likely to blow up.
+
+Output: **$y$**, difficulty parameter. Difficulty is $2^y$. Amounts are in sats.
```

Comment:
Don't bold this. Otherwise they look like vectors or matrices in standard math notation

```suggestion
Output: $y$, difficulty parameter. Difficulty is $2^y$. Amounts are in sats.
```